cmd/evm/internal/t8ntool/transition.go

```diff
@@ -300,7 +300,7 @@ func (g Alloc) OnAccount(addr *common.Address, dumpAccount state.DumpAccount) {
 	if addr == nil {
 		return
 	}
-	balance, _ := new(big.Int).SetString(dumpAccount.Balance, 10)
+	balance, _ := new(big.Int).SetString(dumpAccount.Balance, 0)
```

Comment:
Original code is 10, now it's 0. Is it a mistake ?

Reply:
Good question. In `SetString` , `10` means "base 10", interpret it as decimal string. Whereas `0` means that it can be either hex or decimal: if `"123012"` it is decimal, if `0x`-prefixed, it's hexadecimal. 
So this change makes it so that if `dumpAccount.Balance` happens to be 0x-prefixed, we handle it ok. 


Previously, it never was, but for a while there, before defaulting to `dec`, we had it in hex. Regardless, this change here makes the code more robust.